crypto_libraries/inc/nx_crypto_ecdh.h

```diff
@@ -65,7 +65,9 @@ extern   "C" {
 
 
 /* Max Elliptic-curve Diffie-Hellman key size. Buffer size for calculations is 4X the key size */
-#define NX_CRYPTO_ECDH_MAX_KEY_SIZE     (68)
+#ifndef NX_CRYPTO_ECDH_MAX_KEY_SIZE
+#define NX_CRYPTO_ECDH_MAX_KEY_SIZE     (80)  /* Buffer size is increased to accomodate OEM keys */
```

Comment:
this should be the original value of 68. 
The idea is that their code does not change except for the ifndef. 
The user i.e FSP can redefine that macro in the nx_user.h since FSP generates the nx_user.h and that file is included here

Reply:
also remove the "/* Buffer size is increased to accomodate OEM keys */" comment

Reply:
updated.